sdks/python/apache_beam/transforms/combiners.py

```diff
@@ -282,6 +282,9 @@ def _sort_buffer(self, buffer, lt):
       buffer.sort(cmp=lambda a, b: (not lt(a, b)) - (not lt(b, a)),
                   key=self._key_fn)
 
+  def display_data(self):
+    return {'n': self._n}
```

Comment:
I did not add `reverse` because `TopCombineFn` is not used by itself, instead using `Largest`, `Smallest`, `LargestPerKey`, `SmallestPerKey`. These classes are shown as display data, so if you do `combiners.Smallest(5)`, it uses a `TopCombineFn(5, reverse=True)` internally. I felt it would be confusing to have both the class `combiners.Smallest`, and another dd item `reverse` with value `True` as distinct display data items. (sorry for the wall of text)

As for `compare`, I'm not sure, since it seems it's usually a lambda. @aaltay do you think we should have the `compare` function as display data? Maybe only if it's a named function... what do you think?